releasenotes/notes/django-request-uri-safe-guards-fdd8bd664374ded1.yaml

```diff
@@ -1,4 +1,4 @@
 ---
 fixes:
   - |
-    Add safe guards for building Django http.url span tag
+    django: add safe guards for building http.url span tag
```

Comment:
I noticed the other release notes end with periods but this one does not

Reply:
Good catch! Looks like we have quite a few others that don't as well